content/applications/productivity/data_cleaning.rst

```diff
@@ -328,10 +328,145 @@ Click on the :icon:`fa-bars` :guilabel:`Records` smart button to manage these re
 Cleaning`), Odoo suggests formatting changes to data in fields of a record, to follow a convention
 set by the field cleaning rules.
 
+.. image:: data_cleaning/data-cleaning-field.png
+   :align: center
+   :alt: Field Cleaning Records dashboard in the Data Cleaning application.
+
+The :guilabel:`CLEANING RULE` sidebar lists each of the active cleaning rules, and displays the
+total number of records detected beside each rule.
+
+By default, the :guilabel:`All` rule is selected. Records are displayed in the list view, with the
+following columns:
+
+- :guilabel:`Record ID`: the ID of the original record.
+- :guilabel:`Record Name`: the name or title of the original record.
+- :guilabel:`Field`: the original record's field that contains the value to format.
+- :guilabel:`Current`: the current value in the field of the original record.
+- :guilabel:`Suggested`: the suggested formatted value in the field of the original record.
+
+To clean and format records, click on the :icon:`fa-check` :guilabel:`Validate` button on the row of
+the record.
+
+Upon doing so, the record is formatted and/or cleaned.
+
+.. tip::
+   Discard records by clicking the :icon:`fa-times` :guilabel:`Discard` button. Upon doing so, the
+   record is hidden from the list and won't be detected by the field cleaning rule again in the
+   future.
+
+   View discarded records by selecting the :guilabel:`Discarded` filter from the :ref:`search bar
+   <search/filters>`.
+
+.. _data_cleaning/field-cleaning-rule:
+
+Field cleaning rules
+--------------------
+
+The :guilabel:`Field Cleaning Rules` page (:menuselection:`Data Cleaning app --> Configuration -->
+Rules: Field Cleaning`) is where the conditions for fields to be cleaned and formatted can be set.
+
+These rules can be configured for each model in the database, and with varying levels of
+specificity.
+
+.. tip::
+   The field cleaning rules run once every day, by default, as part of a scheduled action chron
+   (*Data Cleaning: Clean Records*). However, each rule can be :ref:`ran manually
+   <data-cleaning/run-field-cleaning-rule>` anytime.
+
+By default, a :guilabel:`Contact` rule exists to format and clean up the *Contacts* app records.
+Select the :guilabel:`Contact` record to make edits, or select the :guilabel:`New` button to create
+a new rule.
+
+On the field cleaning rule form, first choose a :guilabel:`Model` for this rule to target. Selecting
+a model updates the rule title to the chosen model.
+
+Next, configure at least one rule by clicking :guilabel:`Add a line` in the :guilabel:`Rules`
+section.
+
+Upon doing so, a :guilabel:`Create Rules` popover window appears with the following fields to
+configure:
+
+- :guilabel:`Field To Clean`: select a field from the model to assign to an action.
+- :guilabel:`Action`: choose one of the following actions:
+
+  - :guilabel:`Trim Spaces`: reveals the :guilabel:`Trim` field to select the :guilabel:`All Spaces`
```

Comment:
For each of the Actions, I think you can remove the colon. Since they are all written as a complete sentence, the colon almost seems unnecessary (and as if you are breaking up a sentence). Completely optional, but I think they work without them.

":guilabel:`Trim Spaces` reveals the :guilabel:`Trim` field to select the :guilabel:`All Spaces`..."